common/src/main/java/whocraft/tardis_refined/common/block/device/AstralManipulatorBlock.java

```diff
@@ -41,9 +38,31 @@ public InteractionResult use(BlockState blockState, Level level, BlockPos blockP
 
                         astralManipulatorBlockEntity.OnRightClick(itemStack);
                     }
+
```

Comment:
Make this a util method in a different class, commented code shouldn't exist in the repository 
We could wrap it in a isDev check - I think our mod loaders supply something to check

Reply:
Found `Platform.isProduction()`